server/routes/articleRoute.js

```diff
@@ -11,39 +11,39 @@ import {
   from '../middleware/validator';
 
 const router = express.Router();
-const articleController = new ArticleController();
-const commentController = new CommentController();
+const { createArticle } = ArticleController;
+
 
 router.post('/articles',
   isValidContentType,
   isEmployee,
   isArticleReqValid,
-  articleController.createArticle);
+  createArticle);
 
-router.patch('/articles/:articleId',
-  isValidContentType,
-  isEmployee,
-  isArticleReqValid,
-  isTheOwner,
-  articleController.editArticle);
+// router.patch('/articles/:articleId',
+//   isValidContentType,
+//   isEmployee,
+//   isArticleReqValid,
+//   isTheOwner,
+//   articleController.editArticle);
```

Comment:
Please remove all commented lines of code from your PR

Reply:
Removed